back/tests/cmd/openbsrv-integration/integration_test.go

```diff
@@ -34,38 +35,100 @@ func TestClientServices(t *testing.T) {
 
 func userSvcAddAndFndRoleFn(ctx context.Context, conn *grpc.ClientConn, clnt pb.UserSvcClient) func(*testing.T) {
 	return func(t *testing.T) {
-		want := "testRole"
+		tests := []struct {
+			want   string
+			addReq *pb.AddRoleReq
+			fndReq *pb.FndRolesReq
+		}{
+			{
+				"user",
+				&pb.AddRoleReq{Name: "user"},
+				&pb.FndRolesReq{
+					RoleIds:   []string{},
+					RoleNames: []string{"user"},
+					Limit:     100,
+					Lapse:     0,
+				},
+			},
+			{
+				"admin",
+				&pb.AddRoleReq{Name: "admin"},
+				&pb.FndRolesReq{
+					RoleIds:   []string{},
+					RoleNames: []string{"admin"},
+					Limit:     100,
+					Lapse:     0,
+				},
+			},
+		}
 
-		_, err := clnt.AddRole(ctx, &pb.AddRoleReq{Name: want})
-		if err != nil {
-			t.Fatal(err)
+		for _, tt := range tests {
+			_, err := clnt.AddRole(ctx, tt.addReq)
+			if err != nil {
+				t.Fatal(err)
+			}
+
+			r, err := clnt.FndRoles(ctx, tt.fndReq)
+
+			if len(r.Items) != 1 {
+				t.Fatalf("got: no items, want: %s", tt.want)
+			}
```

Comment:
Please check explicitly for 0 and >1. >1 may occur if the database is not cleared between runs. It's probably not likely, but might as well be clear here.